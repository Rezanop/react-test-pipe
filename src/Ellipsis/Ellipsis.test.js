import { Ellipsis } from '..'
import React from 'react'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(<Ellipsis />).toJSON()
  expect(tree).toMatchSnapshot()
})
