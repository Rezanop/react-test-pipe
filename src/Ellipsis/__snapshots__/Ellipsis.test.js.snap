// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <style />,
  <div
    className="lds-ellipsis"
  >
    <div />
    <div />
    <div />
    <div />
  </div>,
]
`;
